File: Platform/codesignal/Intro/34.extractEachKth/solution.js
function solution(arr, k) {
  return arr.filter((_, i) => (i+1) % k != 0)
}

function solution(arr, k) {
  return arr.filter((_, i) => (i+1) % k)
}

console.log(solution([1, 2, 3, 4, 5, 6, 7, 8, 9, 10], 3));

File: Platform/codesignal/Intro/34.extractEachKth/solution.py
def solution(arr, k):
  return [ arr[i] for i in range(len(arr)) if (i + 1) % k != 0]

def solution(a, k):
  return [x for i, x in enumerate(a) if (i+1)%k !=0]

def solution(arr, k):
    del arr[k-1::k]
    return arr


print(solution([1, 2, 3, 4, 5, 6, 7, 8, 9, 10], 3))